### PR TITLE
Adding README, images, and CONTRIBUTING

### PR DESCRIPTION
**Pull request by dkosteck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+      # Will result in {'lab': {'switches': {'ip': {'name: 'switch_name', 'location': 'location', 'username': 'username', 'password': 'password', type: 'type'}, {'ip_2':...}}}
+
+```
+    - For additional usage options see the help option of the script.
+6. If the script was run successfully, go to the GLPI URL and ensure that fields were correctly populated.
+7. If desired, manually set the location, technician, group, status, and any other desired fields for this machine, as these fields may help with identification or reporting later.
+8. If there was previously information populated, check fields are remove redundant information if present(for example, in the 'Components' section one may need "Unspecified" memory if previously populated). This has been specifically noticed as a remnant of running the Redfish based script, and one work around is currently to simply remove the machine if populating it a second time.
+
+### OS Agnostic, IPMI and Redfish based workflow (Remote):
+![alt text](/docs/images/GLPI-Remote-REST.png)
```
Comment (adhil0) [inline README.md:73]: Nitpicky, but the script wouldn't necessarily be run on a jumphost. Maybe replace with a more general term, like "Local Machine"?

**Review by adhil0 — CHANGES_REQUESTED** [review]
Marked a couple of nits for discussion, but overall i think it looks really good. The diagrams make it way easier to understand which script to use in each situation.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Contributing to ate-lab-management
+
+All contributions are valued and welcomed, whether they come in the form of code, documentation, ideas or discussion.
+While we have not applied a formal Code of Conduct to this, or related, repositories, we require that all contributors
+conduct themselves in a professional and respectful manner.
+
+## Issues
+
+The easiest way to contribute to ate-lab-management is through Issues. This could be by making a suggestion, reporting a
```
[the remark this answers is not in the document]
Comment (dkosteck) [reply CONTRIBUTING.md]: @adhil0 Please check the most recent commit, should be updated

File: README.md (hunk 1 above)
Comment (dkosteck) [reply README.md:73]: @adhil0 Please check the most recent commit, figure should be updated

**Review by adhil0 — APPROVED** [review]
Thanks for the changes, looks great

**Pull request merged**